transpiler/src/target/typescript/core/components/repo/helpers/mongo/aggregateCrudRepo.ts

```diff
@@ -1,43 +1,69 @@
 import { IntermediateASTTree } from '../../../../../../../ast/core/intermediate-ast/IntermediateASTTree.js';
-import { DomainCreateParameterTypeNode } from '../../../../../../../ast/core/intermediate-ast/nodes/Domain/DomainCreateParameterTypeNode.js';
+import { DomainCreateParameterNode } from '../../../../../../../ast/core/intermediate-ast/nodes/Domain/DomainCreateParameterNode.js';
+import { PropsNode } from '../../../../../../../ast/core/intermediate-ast/nodes/Props/PropsNode.js';
 import { ValueObjectDeclarationNode } from '../../../../../../../ast/core/intermediate-ast/nodes/valueObject/ValueObjectDeclarationNode.js';
 import { BitloopsTypesMapping } from '../../../../../../../helpers/mappings.js';
-import { isVO } from '../../../../../../../helpers/typeGuards.js';
-import { TTargetDependenciesTypeScript, TVariable, fieldKey } from '../../../../../../../types.js';
+import { isProps, isVO } from '../../../../../../../helpers/typeGuards.js';
+import {
+  TTargetDependenciesTypeScript,
+  TVariable,
+  fieldKey,
+  fieldsKey,
+} from '../../../../../../../types.js';
 import { getChildDependencies } from '../../../../dependencies.js';
 import { modelToTargetLanguage } from '../../../../modelToTargetLanguage.js';
 
-// TODO TPropsValues where deleted, fix this
+// TODO TPropsValues were deleted, fix this
 type TPropsValues = any;
```

Comment:
I remember adding this TPropsValues type when the file was broken, do we need to remove it now if everything works fine?

Reply:
Or use the correct one, dunno